container/image_test.py

```diff
@@ -216,48 +216,6 @@ def test_with_rfc_3339_creation_time(self):
       cfg = json.loads(img.config_file())
       self.assertEqual('1989-05-03T12:58:12.345Z', cfg.get('created', ''))
 
-  # This test is flaky. If it fails, do a bazel clean --expunge_async and try again
-  def test_with_stamped_creation_time(self):
```

Comment:
Why were these tests deleted?

Reply:
The primary reason for making this PR is that the TarFileWriter in rules_pkg now preserves the mtimes on files in input tars by default (instead of overwriting them to different values based on the presence of `--stamp`), so the default behavior described in these tests no longer applies, if I'm understanding them correctly. @smukherj1 has some extra context on this.

Reply:
These tests validate the image creation time populated in the image config. See [here](https://github.com/bazelbuild/rules_docker/blob/634ff5b5678e0db01baef8fadc9bdb2493193aee/container/image.bzl#L235). This should be unaffected by this change because the image config is created by the Go binary [here](https://github.com/bazelbuild/rules_docker/blob/634ff5b5678e0db01baef8fadc9bdb2493193aee/container/go/cmd/create_image_config/create_image_config.go), not the layer tarball generator that your change affects. Can you look into why the creation time of the image is changing as well?